src/test/java/org/commcare/formplayer/tests/CaseClaimTests.java

```diff
@@ -121,7 +121,8 @@ public void testQueryScreen() throws Exception {
                 QueryResponseBean.class);
         assert queryResponseBean.getDisplays()[0].getValue().contentEquals("Burt");
         assertArrayEquals(queryResponseBean.getDisplays()[1].getItemsetChoices(), new String[]{"karnataka", "Raj as than"});
-        assertArrayEquals(queryResponseBean.getDisplays()[2].getItemsetChoices(), new String[]{"Bangalore", "Hampi"});
+        assertArrayEquals(queryResponseBean.getDisplays()[2].getItemsetChoices(), new String[]{"Bangalore", "Baran", "Hampi", "Kota"});
```

Comment:
not quite following this test. How come there are more choices now?

Reply:
Would it be useful to add a line to test the value of `queryResponseBean.getDisplays()[2].value`?

Reply:
choices for this field (`district`) filters on the selection for the `state` field. Since with the multiselect, we have changed state selection to add one more state, the district options have increased as well. 

Added tests on `value` . Thanks for the prompt.